.env.example

```diff
@@ -26,6 +26,15 @@ IAB_KOVAN_ADDRESS=address_iab_kovan
 IAB_MAINNET_ADDRESS=0xb015D9bBabc472BBfC990ED6A0C961a90a482C57
```

Comment:
#### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Reply:
you are right merging both PR.